pkgs/os-specific/bsd/freebsd/pkgs/mkDerivation.nix

```diff
@@ -104,7 +104,7 @@ lib.makeOverridable (
           rhelmot
           artemist
         ];
-        platforms = lib.platforms.unix;
+        platforms = lib.platforms.freebsd;
```

Comment:
Marking these as unix by default was intentional - we automatically add a compat layer to make stuff work on linux when possible. What did you run into?

Reply:
That's fair. If you want to change this default, can you take all the freebsd derivations that _did_ build successfully for linux and mark them as platforms = unix?